### ISSUE TITLE
[Feature Request] - Only reload on page "click" - not Tab focus

### ISSUE THREAD

**Issue by DP-M** [rNeomy/auto-tab-discard#41]
I routinely have 20-100 tabs open (using tree style tabs over multiple virtual screens), and when going through doing "mass closings", the tab focus often changes to a tab I don't necessarily want to reload, but instead plan on closing. This results in slowing down firefox as I do lots of reloads, before I can close them. 

I previously used "the great extender" in chrome, which would display a (poor, but good enough) screenshot of the page, and only reload if I clicked the screenshot. this was MUCH less resource intensive than doing a reload of the tab any time I accidentally made it active. 

Thanks!

**Comment by derekbasch** [comment]
+1 for this feature request. I have the same issue.

**Comment by gilesbutler** [comment]
I just switched from Chrome with [The Great Suspender](https://chrome.google.com/webstore/detail/the-great-suspender/klbibkeccnjlkjkiokjodocebajanakg?hl=en) too. It definitely feels like a faster experience when the tab doesn't automatically reload.

I find the most annoying thing in FF is when I close a tab and it goes to the previous tab, Auto-Tab-Discard will reload it. 99% of the time I don't want it to reload as I was just closing the last tab.

Apart from that, awesome plugin thanks @rNeomy

**Comment by rNeomy** [comment]
This extension uses the native method to discard tabs. Unfortunately, there is no control on how tabs are loaded. You may request this from browser developers.

the great extender extension replaces the tab with an internal page. This method is definitely inefficient but has a better control on how tabs get reloaded.

As a workaround we have three new buttons in the popup UI;
![screen shot 2018-05-23 at 2 58 46 pm](https://user-images.githubusercontent.com/14104614/40419249-3b7b2af8-5e74-11e8-8128-8ad1ddde2f61.png)

**Comment by foxpup** [comment]
I too would like this feature. :-)

**Comment by lincolnthomas** [comment]
Hi @rNeomy,
Great workaround! Just what I needed.
Can you add the Close button to the existing Auto Tab Discard context menu of a tab? Or add an option to make this the default behavior of closing a tab? I see that if there are no other non-discarded tabs it says "No active tab is present", but in that case it could just follow the browser's default behavior to pick another tab, which will re-open it.
Then this would be a real fix instead of a workaround!

**Comment by rNeomy** [comment]
@lincolnthomas 

> Can you add the Close button to the existing Auto Tab Discard context menu of a tab

The menu is limited to 6 items. Unless we have a selection tool in the options page, we cannot add more items.

> it could just follow the browser's default behavior to pick another tab, which will re-open it.

Fixed!

**Comment by laertis** [comment]
That'd be great and would really lift of the experience when using this extension.
I'll check if can create a brunch with the feature if I have some spare time

**Comment by milkmiruku** [comment]
I have submitted an issue regarding this behaviour on the Firefox Bugzilla - https://bugzilla.mozilla.org/show_bug.cgi?id=1505960

**Comment by noblepirate** [comment]
+ 1000000 for this request!

**Comment by RobertAb1** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1554482
https://github.com/rNeomy/auto-tab-discard/issues/87

**Comment by mikaelstaer** [comment]
Another vote for this!

**Comment by ajck** [comment]
+1 for this. Super annoying when a tab reactivates/reloads when you didn't want it to!
@rNeomy any chance a tab could close somehow without reloading any of the previously discarded tabs?
Many thanks for a great extension otherwise.

**Comment by friendlytherapist** [comment]
This is all the extension is missing!

**Comment by mxmilkiib** [comment]
N.b. I don't think there is any advantage to bumping this issue rather than the one on the Firefox bugzilla.

**Comment by vvkdev** [comment]
Can't use this new extension due to the lack of this feature (((
Have to use the old "Tab Suspender" version and waiting for...
There is the option "When discarding, replace the page with a local dummy alternative (beta)"  but in Chrome it is greyed ((
Do you have plans to activate this option?
Thank you!

**Comment by maanemann** [comment]
> Can't use this new extension due to the lack of this feature ((( Have to use the old "Tab Suspender" version and waiting for... There is the option "When discarding, replace the page with a local dummy alternative (beta)" but in Chrome it is greyed out (( Do you have plans to activate this option? Thank you!

Yes! Also greyed out in firefox. This was actually the single feature i downloaded the plugin for. To still have the tab open, but replaced with a dummy image, so it's visually represented when i'm overlooking my open windows to remember what was going on. And so we prevent that annoying problem of reactivating unintentionally others were talking about. I can't find any suspender plugin that does that, except the long discarded 'great suspender'!

**Comment by ilhamprabuzakys** [comment]
Any update? this thread was old a feature request.

But there's no update yet

**Comment by Zalext** [comment]
> This extension uses the native method to discard tabs. Unfortunately, there is no control on how tabs are loaded. You may request this from browser developers.
> 
> the great extender extension replaces the tab with an internal page. This method is definitely inefficient but has a better control on how tabs get reloaded.

Hi,
I was using your old Tab-Unloader Extension til yesterday since started crashing on Vivaldi 7.0.
That allows to choose the default or etension's way to hibernate.
Also provides that requested option.

Would be posible to update that other for those interested on the "Click to Awake"?

Thank you
